src/java.base/share/classes/sun/security/ssl/ECDHKeyExchange.java

```diff
@@ -202,8 +200,8 @@ void checkConstraints(AlgorithmConstraints constraints,
                         "ECPublicKey does not comply to algorithm constraints");
                 }
             } catch (GeneralSecurityException | java.io.IOException e) {
-                throw (SSLHandshakeException) new SSLHandshakeException(
-                        "Could not generate ECPublicKey").initCause(e);
+                throw new SSLHandshakeException(
+                        "Could not generate ECPublicKey", e);
```

Comment:
Nit:  I think combining these lines would be < 80 chars

Reply:
The exception name is too long to have in one line.  There are 83 characters in total if combining line 203 and 204.  3 characters exceeding 80 chars per line limit is not easy to tell with eyes.

Reply:
The 80 character recommendation isn't a hard limit, I favor readability in the 80-100 char range.